### ISSUE TITLE
Consider adding formattting datetime as relative?

### ISSUE THREAD

**Issue by dimitarvp** [mwhitworth/relative_datetime#17]
Hey,
I haven't found almost any libraries doing what yours does. But in this case I need a library that does exactly the opposite as yours: given a `DateTime` (or `NaiveDateTime`) function argument, returns string like "17 seconds ago" or "1 hour ago" or "3 days ago".

Would you consider adding that to your library?

**Comment by mwhitworth** [comment]
Hi @dimitarvp!

This function exists in [Timex.from_now/{1,2,3}](https://hexdocs.pm/timex/Timex.html#from_now/1), which can format in a relative format, optionally with a given `datetime` and `locale`.

I'd be tempted to include it in this library later - to add another option to "and the kitchen sink" libraries like Timex given this narrow use case - so I'll leave this ticket open once the relative datetime parsing is broadly feature complete.

**Comment by dimitarvp** [comment]
Yep, I know it's there, I simply find `Timex` too heavy a dependency. If you add it at one point that will be great. Thanks for responding. 😃
